### DISCUSSION TITLE
Show changes since last review

### DISCUSSION THREAD

**Discussion by jukzi** [eclipse-jdt/eclipse.jdt.core#3401] [Show and tell]
Just found this Option:
![image](https://github.com/user-attachments/assets/d4fcc190-2e5a-4fcd-932a-01fcbf58f6fa)
which may help in code review @stephan-herrmann @iloveeclipse

**Comment by stephan-herrmann** [comment]
>  which may help in code review @stephan-herrmann @iloveeclipse

it certainly helps if those changes don't include a rebase :)
